gslib/utils/text_util.py

```diff
@@ -174,19 +181,19 @@ def FixWindowsEncodingIfNeeded(input_str):
   to Unicode.
 
   Args:
-    input_str: (str) The input string.
+    input_str: (str or bytes) The input string.
```

Comment:
"unicode or bytes" instead?  The return type is listed as "unicode", so I'd like to be consistent at least within the same docstring.